### PR TITLE
Add styling to bookstore app (PR)

### PR DESCRIPTION
**Pull request by teke85**
## In this project milestone, the following requirements were implemented:

- You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project).
- Open the project Bookstore CMS.
- Add any missing JSX elements to your projects (without adding extra functionality)
- You may use the CSS rules detailed in Zeplin to style the website.
- Your final design should match Zeplin's design (this includes all elements + the same styling)

### CODE REVIEW

File: src/Components/AddBook.js [hunk 1]
```diff
@@ -30,27 +29,32 @@ const AddBook = () => {
...
+          className="input"
+          type="text"
+          placeholder="Title"
+          name="title"
+          required
+          value={newBook.title}
+          onChange={handleInput}
+        />
+        <select className="input" style={{ width: '165px' }}>
+          <option value="" disabled>
```
Comment (EricMbouwe) [inline src/Components/AddBook.js]: - [ ] No inline styling (Note: React inline styling is allowed but a variable should be created and passed to React style. Let's fix that
Here is an example

```js
const myComponentStyle = {
  color: 'blue',
  lineHeight: 10,
  padding: '1.5em',
}

function MyComponent(){
    return <div style={myComponentStyle}> Inline Styled Component</div>
}
```

**Review by EricMbouwe — CHANGES_REQUESTED** [review]
Hi @teke85, 

Good job so far 🔥!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🔥🎉
- PR ✔
- No linter errors ✔
- Css best practices ✔

# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Components/AddBook.js (hunk 1 above)
Comment (teke85) [reply src/Components/AddBook.js]: Hi @EricMbouwe thank you for your feedback. I have removed the inline styling

**Review by victorokoroji — APPROVED** [review]
Hi @teke85, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the Git Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**